modules/mobile/app/controllers/mobile/v0/maintenance_windows_controller.rb

```diff
@@ -15,7 +15,7 @@ class MaintenanceWindowsController < ApplicationController
         %i[bgs payment_history],
         %i[arcgis facility_locator],
         %i[caseflow appeals],
-        %i[vet360 military_service_history],
```

Comment:
confirmed with Experience team that this is a replacement for vet360. vet360 is too broad and never goes down as a whole, only different APIs within it.